CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ IF(NOT KOKKOS_HAS_TRILINOS)
   endif()
   include(${Kokkos_BINARY_DIR}/kokkos_generated_settings.cmake)
   install(FILES ${Kokkos_BINARY_DIR}/kokkos_generated_settings.cmake DESTINATION lib/cmake/Kokkos)
+  install(FILES ${Kokkos_BINARY_DIR}/kokkos_generated_settings.cmake DESTINATION ${CMAKE_INSTALL_PREFIX})
```

Comment:
Putting it one more place doesn't hurt I guess. This will make it consistent with generate_makefile installs.